Makefile

```diff
@@ -435,7 +435,11 @@ test: lint karma
 
 .PHONY: runserver
 runserver: remove_old_config
-	$(REPO_BIN_DIR)/run-server.sh
+	@if [[ -z "${NO_WATCH}" ]]; then \
+	  $(REPO_BIN_DIR)/run-server.sh; \
+	else \
+	  node bin/server.js; \
+	fi
```

Comment:
yeah, lets go with that - I was originally thinking of keeping `make runserver` so that is was compatible with loop-client, but I can just add another target there whilst we transition.